test/combinatorics.jl

```diff
@@ -1,18 +1,35 @@
-@test factorial(7) == 5040
-@test factorial(7,3) == 7*6*5*4
 @test binomial(5,3) == 10
+@test binomial(2,1) == 2
+@test binomial(1,2) == 0
+@test binomial(-2,1) == -2
```

Comment:
I guess this makes sense, but I found it surprising. Add a comment to make sure someone doesn't delete it out of puzzlement later?